README.md

```diff
@@ -22,6 +22,10 @@ in the `env` section of your
 to use a different major (e.g., "10" or "9.6") version. This feature
 is experimental and subject to change.
 
+## PostgreSQL fsync
+
+If you want to turn off fsync during your test, you can set `HEROKU_CI_POSTGRES_FSYNC_OFF` in your [app.json env](https://devcenter.heroku.com/articles/heroku-ci#environment-variables-env-key) section.
```

Comment:
```suggestion
If you want to turn off fsync during your test, you can set `HEROKU_CI_POSTGRES_FSYNC_OFF` in your [app.json env](https://devcenter.heroku.com/articles/heroku-ci#setting-environment-variables-the-env-key) section.
```

Reply:
I noticed the link doesn't link to the correct place any more. Added suggestion above, made a new PR to correct the existing one: https://github.com/heroku/heroku-buildpack-ci-postgresql/pull/37.